doc/specs/stdlib_math.md

```diff
@@ -91,6 +91,52 @@ program demo_clip_real
 end program demo_clip_real
 ```
 
+### `gcd` function
+
+#### Description
+
+Returns the greatest common divisor of two integers.
+
+#### Syntax
+
+`res = [[stdlib_math(module):gcd(interface)]] (a, b)`
+
+#### Status
+
+Experimental
+
+#### Class
+
+Elemental function.
+
+#### Argument(s)
+
+`a`: An integer.
+`b`: Another integer.
```

Comment:
This part needs to be improved a bit.
You can mention `intent` here. If possible try to make it similar to other procedures mentioned in this doc to be consistent.

Reply:
I am not sure what to write for the a and b. Don't want to repeat the description of the function.

I thought intent would be read from the code itself.